addons/sms/tests/test_sms_template_access_rights.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests.common import SavepointCase, new_test_user
+from odoo.exceptions import AccessError
+
+
+class TestSmsTemplateAccessRights(SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.sms_manager = new_test_user(cls.env, login='sms_manager', groups='base.group_system')
+        cls.sms_user = new_test_user(cls.env, login='sms_user', groups='base.group_user')
```

Comment:
You could search your models here.

`cls.sms_enabled_models = self.env['ir.model'].search([('is_mail_thread_sms', '=', True), ('transient', '=', False)])`
(No need for the &, it's the default).

Reply:
done